torch/__init__.py

```diff
@@ -1285,6 +1289,96 @@ def set_default_dtype(d: "torch.dtype", /) -> None:
     _C._set_default_dtype(d)
 
 
+def set_future_lazy_clone(mode: builtins.bool) -> None:
+    r"""Enables future behavior of always copying, for operators that currently
```

Comment:
I see this documentation as the place where a user will go to understand what the deprecation warning means and what they can do about it, so it will be important to make sure it's clear and makes sense